apps/password-reset/src/app/app.component.ts

```diff
@@ -13,18 +15,22 @@ import { TokenExpiredDialogComponent } from './dialogs/token-expired-dialog/toke
 export class AppComponent implements OnInit{
 
   constructor(private dialog: MatDialog,
-              private usersService: UsersManagerService) {
-  }
+              private usersService: UsersManagerService,
+              private preferredLangService: PreferredLanguageService,
+              private translateService:TranslateService) { }
 
   ngOnInit() {
+    const prefLang = this.preferredLangService.getPreferredLanguage(null);
+    this.translateService.use(prefLang);
+
     const queryParams = location.search.substr(1);
-    let mode = ''
+    let mode = '';
     if (queryParams.includes('activation')) {
-      mode = 'activation'
+      mode = 'activation';
     } else if(queryParams.includes('token')) {
-      mode = 'reset'
+      mode = 'reset';
     } else {
-      mode = 'change'
+      mode = 'change';
```

Comment:
we disscuss with pavel that mode change will not be in password reser app
please remove it

Reply:
I don't think this should be part of this PR as this was about setting language and auth filter from query params.